packages/flutter_tools/test/commands.shard/hermetic/drive_test.dart

```diff
@@ -54,6 +54,7 @@ void main() {
       appStopper = (DriveCommand command) {
         throw 'Unexpected call to appStopper';
       };
+      mockDevice = MockAndroidDevice();
```

Comment:
It looks like some tests below will replace this with a `MockDevice()` rather than a `MockAndroidDevice()`

Reply:
Hm.  I'll move this down to the test that needs it and check if there are any other tests using it but not initializing it.

Reply:
Done - no other tests are accessing this object without setting it up first.